app/javascript/src/LoginPage/utils/formValidation.js

```diff
@@ -0,0 +1,96 @@
+import validate from 'validate.js'
+
+const constraintsTypes = {
+  text: {
+    presence: true,
+    length: {minimum: 1}
+  },
+  email: {
+    presence: true,
+    email: true,
+    length: {minimum: 1}
+  },
+  password: {
+    presence: true,
+    length: {minimum: 1}
```

Comment:
Maybe minimum 6 like in other forms?

Reply:
In fact I don't know what is the minimum length for this specific form, because of that I leave it with just one.

Reply:
Can be 6 or 8, depending on the provider:
https://github.com/3scale/porta/blob/master/app%2Fmodels%2Fuser.rb#L80
https://github.com/3scale/porta/blob/master/app%2Fmodels%2Fuser.rb#L105-L106

I suggested using 6 as it is the "standard".

Reply:
Just check what our current minimum is at 3scale.net and stick with it. It should be exactly the same as the Ruby validations

Reply:
ah sorry it depends on settings, so should ideally use those settings to provide the correct feedback.